packaging/linux/deb/package_binaries.sh

```diff
@@ -32,7 +32,9 @@ elif [ "$mode" = "prerelease" ] ; then
   repo_url="http://prerelease.keybase.io/deb"
   # "psmisc" provides "killall", which is used in run_keybase and
   # post_install.sh.
-  dependencies="Depends: libappindicator1, fuse, libgconf-2-4, psmisc"
+  # lsof used in post_install.sh
+  # systemd-container provides machinectl, which is used in post_install.sh
+  dependencies="Depends: libappindicator1, fuse, libgconf-2-4, psmisc, lsof, systemd-container"
```

Comment:
Do you think we'll run into situations where someone wants to turn off systemd support, but this forces them to include the package?  Not sure if we care about that, maybe we're moving towards a systemd-only world anyway?

Reply:
Yeah I was worried about this exactly. I'd like to do a "systemd-container only if systemd is installed" but I doubt it's possible. In either case I think it should be possible to restart for non-systemd too.

I'm just not sure how to figure out whether they are using systemctl to run keybase without systemd-container. `su` actually does work if you pass in `XDG_RUNTIME_DIR` and `DBUS_SESSION_BUS_ADDRESS` (those are set by systemd I believe), but the only way to know that for a user is to use `machinectl`...